scenarios/alert/dotnet/control_tower/AlertSender.cs

```diff
@@ -0,0 +1,29 @@
+﻿using alert_message;
+using MQTTnet.Client;
+using MQTTnet.Client.Extensions;
+
+namespace control_tower
+{
+    internal class AlertSender
+    {
+        IMqttClient _mqttClient;
+        IMessageSerializer _serializer;
+        string _alertTopic = "vehicles/weather/alert";
+
+        public AlertSender(IMqttClient mqttClient)
+        {
+            _mqttClient = mqttClient;
+            _serializer = new Utf8JsonSerializer();
+        }
+
+        public async Task<MqttClientPublishResult> SendAlertAsync(AlertMessage message, CancellationToken ct = default) =>
+            await _mqttClient.PublishAsync(
+                new MQTTnet.MqttApplicationMessageBuilder()
+                   .WithTopic(_alertTopic)
+                   .WithPayload(_serializer.ToBytes(message))
+                   .WithQualityOfServiceLevel(MQTTnet.Protocol.MqttQualityOfServiceLevel.AtLeastOnce)
+                   .WithRetainFlag(false)
+                   .WithMessageExpiryInterval(300)
```

Comment:
Would be better to setup a variable at the class level to highlight this is a choice to select an 'expiration' time of 5 mins (as per the sample document explained).

Reply:
this is a class with a single method, I dont think it's worth it.

Same with the other properties.